diamm/serializers/iiif/service.py

```diff
@@ -121,8 +121,13 @@ def get_folios(self, obj):
 
     def get_composition(self, obj):
         if 'composition_i' in obj:
+            connection = pysolr.Solr(settings.SOLR['SERVER'])
+            q = "pk:{0}".format(obj['composition_i'])
```

Comment:
`q` should generally be "*:*" and the "pk" should be in `fq`. I *think* fq queries are faster, since they are cached; see: http://wiki.apache.org/solr/CommonQueryParameters#fq